[2022-10-09T23:13:41.681+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: cemaco.prep scheduled__2022-08-11T00:00:00+00:00 [queued]>
[2022-10-09T23:13:41.689+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: cemaco.prep scheduled__2022-08-11T00:00:00+00:00 [queued]>
[2022-10-09T23:13:41.689+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-10-09T23:13:41.690+0000] {taskinstance.py:1363} INFO - Starting attempt 4 of 4
[2022-10-09T23:13:41.690+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-10-09T23:13:41.779+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): prep> on 2022-08-11 00:00:00+00:00
[2022-10-09T23:13:41.783+0000] {standard_task_runner.py:54} INFO - Started process 295 to run task
[2022-10-09T23:13:41.785+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'cemaco', 'prep', 'scheduled__2022-08-11T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp4k5nofu8']
[2022-10-09T23:13:41.785+0000] {standard_task_runner.py:83} INFO - Job 10: Subtask prep
[2022-10-09T23:13:41.786+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/dag.py
[2022-10-09T23:13:42.283+0000] {task_command.py:384} INFO - Running <TaskInstance: cemaco.prep scheduled__2022-08-11T00:00:00+00:00 [running]> on host 7dcbfa2899a4
[2022-10-09T23:13:42.514+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cemaco
AIRFLOW_CTX_TASK_ID=prep
AIRFLOW_CTX_EXECUTION_DATE=2022-08-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=4
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-11T00:00:00+00:00
[2022-10-09T23:13:42.515+0000] {tasks.py:83} INFO - Reading the last request file for day 09/10/2022
[2022-10-09T23:13:42.523+0000] {tasks.py:62} INFO - Cleaning the column name
[2022-10-09T23:13:42.536+0000] {tasks.py:98} INFO - Filtering for Codes 91792
[2022-10-09T23:13:42.537+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/tasks.py", line 100, in prep
    dataframe.code.isin(config["code"])
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/series.py", line 5025, in isin
    result = algorithms.isin(self._values, values)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/algorithms.py", line 460, in isin
    "only list-like objects are allowed to be passed "
TypeError: only list-like objects are allowed to be passed to isin(), you passed a [str]
[2022-10-09T23:13:42.549+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=cemaco, task_id=prep, execution_date=20220811T000000, start_date=20221009T231341, end_date=20221009T231342
[2022-10-09T23:13:42.637+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 10 for task prep (only list-like objects are allowed to be passed to isin(), you passed a [str]; 295)
[2022-10-09T23:13:42.680+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-10-09T23:13:42.761+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
